.gitignore

```diff
@@ -1 +1,3 @@
 *.pyc
+TODO.txt
```

Comment:
not really sure what these two `.txt` files are but i'm hesitant to add them to the gitignore since they're not really relevant to graspit_commander (like the compiled python code or maybe directories an ide like pycharm automatically creates). 

If you'd like to stop these files from showing up when you run `git status` you can add them to your local git ignore by modifying the file `.git/info/exclude` which is effectively a local gitignore file